src/test/java/org/openrefine/extensions/commons/functions/ExtractFromTemplateTest.java

```diff
@@ -0,0 +1,26 @@
+package org.openrefine.extensions.commons.functions;
+
+import java.util.Arrays;
+import java.util.Properties;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.google.refine.grel.Function;
+
+public class ExtractFromTemplateTest {
+
+    Function function = new ExtractFromTemplate();
+
+    @Test
+    public void testTemplateName() {
+        String wikitext = "{{some template|bar=test}}\n"
+                + "{{foo1|bar={{other template}}}}\n"
+                + "{{foo2| foo3 = not important| bar = second value }}";
+        
+        Object result = function.call(new Properties(), new Object[] {wikitext});
+
+        Assert.assertEquals(result, Arrays.asList("other template" + ": " + "second value"));
```

Comment:
`Arrays.asList("other template" + ": " + "second value")` is the same thing as `Arrays.asList("other template: second value")`, so that is probably not what you meant, right?